### PR TITLE
[Minor] Fix Travis behavior

### PR DESCRIPTION
**Pull request by seojangho**
This commit allows Travis CI to properly execute tests.

- Modified build command to execute tests properly.
- Allow caching `.m2` for faster testing
- Reduce memory consumption while running tests

### CODE REVIEW

**Comment by wonook** [comment]
@seojangho Can we cache our builds as well with this PR? I've provided a link below:

https://docs.travis-ci.com/user/caching

We can add the lines below to cache our maven directories

```
cache:
  directories:
  - $HOME/.m2
```

**Comment by seojangho** [comment]
@wonook Good idea!

**Comment by wonook** [comment]
I'll merge after the tests

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,11 @@
 # .travis.yml
 # For maven builds
 language: java
+script: "mvn test -B && mvn verify -B"
```
Comment (wonook) [inline .travis.yml]: Can we replace this line with `after_script: "mvn verify -B"`?

**Pull request merged**